src/Common/Commands.Common/MetricHelper.cs

```diff
@@ -226,6 +226,16 @@ private void LogExceptionEvent(AzurePSQoSEvent qos)
                 eventProperties.Add("Message", "Message removed due to PII.");
                 eventProperties.Add("StackTrace", qos.Exception.StackTrace);
                 eventProperties.Add("ExceptionType", qos.Exception.GetType().ToString());
+                Exception innerEx = qos.Exception.InnerException;
```

Comment:
I forget, how do we feel about `var`?

Reply:
not sure :)
Having said that... i would use a var when dealing with anonymous types.. in this case i odnt foresee it changeing from Exception, so i explicitly use the type.